app/components/Header/NavBar.js

```diff
@@ -2,4 +2,5 @@ import styled from 'styled-components';
 
 export default styled.div`
   text-align: center;
+  background-color: primaryDark;
```

Comment:
it won't work that way 

<img width="423" alt="Screenshot 2020-03-31 at 19 15 07" src="https://user-images.githubusercontent.com/13777670/78055540-f9f95b00-7383-11ea-8472-8a8d05158bbd.png">

I've not clue how it resolve yet. Need to dig into how themes works inside react-boilerplate.

Reply:
Here is the link to [docs](https://styled-components.com/docs/advanced)

Reply:
Is it fixed ? Do you need any help ?

Reply:
It's fixed :)